pkgs/development/libraries/faudio/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, cmake, SDL2}:
+
+#TODO: tests
+
+stdenv.mkDerivation {
+	name = "faudio";
+  version = "19.19";
+  src = fetchFromGitHub {
+    owner = "FNA-XNA";
+    repo = "FAudio";
+    rev = "19.09";
+    sha256 = "0fagik55jmy3qmb27nhg0zxash1ahfkxphx8m8gs0pimqqrdrd9d";
+  };
+
+	nativeBuildInputs = [cmake SDL2];
+
+  meta = with stdenv.lib; {
+    description = "Reproduce the window's XAudio library";
```

Comment:
I guess «DirectX» is a better reference than «MS Windows», and «window's» is just a typo in this context.